### PR TITLE
Removes safety straps from batons

### PR DESCRIPTION
**Pull request by azzzertyy**
# Document the changes in your pull request
Was made when jamie thought 1 hit batons were a good idea, now those batons no longer 1 hit, but this remains.
Removes it, probably one of the single stupidest features for everyone, antags and sec both.

# Changelog
:cl:  
rscdel: Removed safety straps
/:cl:

### CODE REVIEW

**Comment by Hislyna** [comment]
Wasn't this a council vote? Pretty sure thisll need another council vote to revert.

**Comment by azzzertyy** [comment]
> Wasn't this a council vote? Pretty sure thisll need another council vote to revert.

Dont believe so.
If it was it never would've been added, original PR was WIDELY disliked.

**Comment by Hislyna** [comment]
> > Wasn't this a council vote? Pretty sure thisll need another council vote to revert.
> 
> Dont believe so. If it was it never would've been added, original PR was WIDELY disliked.

https://forums.yogstation.net/threads/vote-909-test-the-baton-changes-and-hold-a-server-vote-to-dictate-the-path-forward.25377/

**Comment by azzzertyy** [comment]
> > > Wasn't this a council vote? Pretty sure thisll need another council vote to revert.
> > 
> > 
> > Dont believe so. If it was it never would've been added, original PR was WIDELY disliked.
> 
> https://forums.yogstation.net/threads/vote-909-test-the-baton-changes-and-hold-a-server-vote-to-dictate-the-path-forward.25377/

Can you find the poll that was apparently done after this?
This is a vote to test it for one week too, not a vote to keep it.

**Comment by Hislyna** [comment]
> > > > Wasn't this a council vote? Pretty sure thisll need another council vote to revert.
> > > 
> > > 
> > > Dont believe so. If it was it never would've been added, original PR was WIDELY disliked.
> > 
> > 
> > https://forums.yogstation.net/threads/vote-909-test-the-baton-changes-and-hold-a-server-vote-to-dictate-the-path-forward.25377/
> 
> Can you find the poll that was apparently done after this? This is a vote to test it for one week too, not a vote to keep it.

"Test the new baton changes for 1 week (ending next tuesday evening), then release a server poll for 1 week to gather feedback on the change, the result of which will be binding whether we keep or revert the changes. The poll creation will be discussed with Mqiib in staff-council chat to ensure it's worded fairly."

It is a vote to keep it. I will try to look for the vote however.

**Comment by azzzertyy** [comment]
> > > > > Wasn't this a council vote? Pretty sure thisll need another council vote to revert.
> > > > 
> > > > 
> > > > Dont believe so. If it was it never would've been added, original PR was WIDELY disliked.
> > > 
> > > 
> > > https://forums.yogstation.net/threads/vote-909-test-the-baton-changes-and-hold-a-server-vote-to-dictate-the-path-forward.25377/
> > 
> > 
> > Can you find the poll that was apparently done after this? This is a vote to test it for one week too, not a vote to keep it.
> 
> "Test the new baton changes for 1 week (ending next tuesday evening), then release a server poll for 1 week to gather feedback on the change, the result of which will be binding whether we keep or revert the changes. The poll creation will be discussed with Mqiib in staff-council chat to ensure it's worded fairly."
> 
> It is a vote to keep it. I will try to look for the vote however.

I meant that the council vote was a vote to test it, not the poll.

**Comment by Hislyna** [comment]
> > > > > > Wasn't this a council vote? Pretty sure thisll need another council vote to revert.
> > > > > 
> > > > > 
> > > > > Dont believe so. If it was it never would've been added, original PR was WIDELY disliked.
> > > > 
> > > > 
> > > > https://forums.yogstation.net/threads/vote-909-test-the-baton-changes-and-hold-a-server-vote-to-dictate-the-path-forward.25377/
> > > 
> > > 
> > > Can you find the poll that was apparently done after this? This is a vote to test it for one week too, not a vote to keep it.
> > 
> > 
> > "Test the new baton changes for 1 week (ending next tuesday evening), then release a server poll for 1 week to gather feedback on the change, the result of which will be binding whether we keep or revert the changes. The poll creation will be discussed with Mqiib in staff-council chat to ensure it's worded fairly."
> > It is a vote to keep it. I will try to look for the vote however.
> 
> I meant that the council vote was a vote to test it, not the poll.

" release a server poll for 1 week to gather feedback on the change, the result of which will be binding whether we keep or revert the changes."

That part of the council vote means that they made a vote which is binding under this council vote, so it will still fall under council.

**Comment by azzzertyy** [comment]
> > > > > > > Wasn't this a council vote? Pretty sure thisll need another council vote to revert.
> > > > > > 
> > > > > > 
> > > > > > Dont believe so. If it was it never would've been added, original PR was WIDELY disliked.
> > > > > 
> > > > > 
> > > > > https://forums.yogstation.net/threads/vote-909-test-the-baton-changes-and-hold-a-server-vote-to-dictate-the-path-forward.25377/
> > > > 
> > > > 
> > > > Can you find the poll that was apparently done after this? This is a vote to test it for one week too, not a vote to keep it.
> > > 
> > > 
> > > "Test the new baton changes for 1 week (ending next tuesday evening), then release a server poll for 1 week to gather feedback on the change, the result of which will be binding whether we keep or revert the changes. The poll creation will be discussed with Mqiib in staff-council chat to ensure it's worded fairly."
> > > It is a vote to keep it. I will try to look for the vote however.
> > 
> > 
> > I meant that the council vote was a vote to test it, not the poll.
> 
> " release a server poll for 1 week to gather feedback on the change, the result of which will be binding whether we keep or revert the changes."
> 
> That part of the council vote means that they made a vote which is binding under this council vote, so it will still fall under council.

Yes. But i cant find any evidence that this public vote... ever happened, closest i've found is a baton change vote thats dated 2 months prior to this council vote.
Which means it should happen now, and i propose a quickfire vote for it.

**Comment by Hislyna** [comment]
> > > > > > > > Wasn't this a council vote? Pretty sure thisll need another council vote to revert.
> > > > > > > 
> > > > > > > 
> > > > > > > Dont believe so. If it was it never would've been added, original PR was WIDELY disliked.
> > > > > > 
> > > > > > 
> > > > > > https://forums.yogstation.net/threads/vote-909-test-the-baton-changes-and-hold-a-server-vote-to-dictate-the-path-forward.25377/
> > > > > 
> > > > > 
> > > > > Can you find the poll that was apparently done after this? This is a vote to test it for one week too, not a vote to keep it.
> > > > 
> > > > 
> > > > "Test the new baton changes for 1 week (ending next tuesday evening), then release a server poll for 1 week to gather feedback on the change, the result of which will be binding whether we keep or revert the changes. The poll creation will be discussed with Mqiib in staff-council chat to ensure it's worded fairly."
> > > > It is a vote to keep it. I will try to look for the vote however.
> > > 
> > > 
> > > I meant that the council vote was a vote to test it, not the poll.
> > 
> > 
> > " release a server poll for 1 week to gather feedback on the change, the result of which will be binding whether we keep or revert the changes."
> > That part of the council vote means that they made a vote which is binding under this council vote, so it will still fall under council.
> 
> Yes. But i cant find any evidence that this public vote... ever happened, closest i've found is a baton change vote thats dated 2 months prior to this council vote. Which means it should happen now, and i propose a quickfire vote for it.

Sadly as per the codex, you cannot change anything that is council rulled without a council vote. Yogspolitics at its finest

**Comment by azzzertyy** [comment]
> > > > > > > > > Wasn't this a council vote? Pretty sure thisll need another council vote to revert.
> > > > > > > > 
> > > > > > > > 
> > > > > > > > Dont believe so. If it was it never would've been added, original PR was WIDELY disliked.
> > > > > > > 
> > > > > > > 
> > > > > > > https://forums.yogstation.net/threads/vote-909-test-the-baton-changes-and-hold-a-server-vote-to-dictate-the-path-forward.25377/
> > > > > > 
> > > > > > 
> > > > > > Can you find the poll that was apparently done after this? This is a vote to test it for one week too, not a vote to keep it.
> > > > > 
> > > > > 
> > > > > "Test the new baton changes for 1 week (ending next tuesday evening), then release a server poll for 1 week to gather feedback on the change, the result of which will be binding whether we keep or revert the changes. The poll creation will be discussed with Mqiib in staff-council chat to ensure it's worded fairly."
> > > > > It is a vote to keep it. I will try to look for the vote however.
> > > > 
> > > > 
> > > > I meant that the council vote was a vote to test it, not the poll.
> > > 
> > > 
> > > " release a server poll for 1 week to gather feedback on the change, the result of which will be binding whether we keep or revert the changes."
> > > That part of the council vote means that they made a vote which is binding under this council vote, so it will still fall under council.
> > 
> > 
> > Yes. But i cant find any evidence that this public vote... ever happened, closest i've found is a baton change vote thats dated 2 months prior to this council vote. Which means it should happen now, and i propose a quickfire vote for it.
> 
> Sadly as per the codex, you cannot change anything that is council rulled without a council vote.

I dont think you understand what im saying.
This council vote suggests that it would be tested for one week, and then a public vote would be held as to whether or not the change should remain.
As far as i can tell, this second vote NEVER happened.
Council did not do what they were supposed to, and the change has now remained until now, with no public vote being made.
My suggestion is to remedy this failure to comply with councils own votes by making a quickfire vote.

**Comment by azzzertyy** [comment]
Council never ruled on this staying in the game.
Council ruled that a public vote will be held as to whether this should stay in the game.
This was never done.

**Comment by Hislyna** [comment]
> > > > > > > > > > Wasn't this a council vote? Pretty sure thisll need another council vote to revert.
> > > > > > > > > 
> > > > > > > > > 
> > > > > > > > > Dont believe so. If it was it never would've been added, original PR was WIDELY disliked.
> > > > > > > > 
> > > > > > > > 
> > > > > > > > https://forums.yogstation.net/threads/vote-909-test-the-baton-changes-and-hold-a-server-vote-to-dictate-the-path-forward.25377/
> > > > > > > 
> > > > > > > 
> > > > > > > Can you find the poll that was apparently done after this? This is a vote to test it for one week too, not a vote to keep it.
> > > > > > 
> > > > > > 
> > > > > > "Test the new baton changes for 1 week (ending next tuesday evening), then release a server poll for 1 week to gather feedback on the change, the result of which will be binding whether we keep or revert the changes. The poll creation will be discussed with Mqiib in staff-council chat to ensure it's worded fairly."
> > > > > > It is a vote to keep it. I will try to look for the vote however.
> > > > > 
> > > > > 
> > > > > I meant that the council vote was a vote to test it, not the poll.
> > > > 
> > > > 
> > > > " release a server poll for 1 week to gather feedback on the change, the result of which will be binding whether we keep or revert the changes."
> > > > That part of the council vote means that they made a vote which is binding under this council vote, so it will still fall under council.
> > > 
> > > 
> > > Yes. But i cant find any evidence that this public vote... ever happened, closest i've found is a baton change vote thats dated 2 months prior to this council vote. Which means it should happen now, and i propose a quickfire vote for it.
> > 
> > 
> > Sadly as per the codex, you cannot change anything that is council rulled without a council vote.
> 
> I dont think you understand what im saying. This council vote suggests that it would be tested for one week, and then a public vote would be held as to whether or not the change should remain. As far as i can tell, this second vote NEVER happened. Council did not do what they were supposed to, and the change has now remained until now, with no public vote being made. My suggestion is to remedy this failure to comply with councils own votes by making a quickfire vote.

The vote was on the server itself, not the discord. I remember voting on it myself (trust me bro), but nonetheless it /should/ be archived somewhere, in which council can drag it out and confirm/deny whatever was on there. Otherwise yes, a vote should be made if it was not.

**Comment by azzzertyy** [comment]
> The vote was on the server itself, not the discord. I remember voting on it myself (trust me bro), but nonetheless it /should/ be archived somewhere, in which council can drag it out and confirm/deny whatever was on there. Otherwise yes, a vote should be made if it was not.

What? Why is a public vote being done on the game?

**Comment by Hislyna** [comment]
> > The vote was on the server itself, not the discord. I remember voting on it myself (trust me bro), but nonetheless it /should/ be archived somewhere, in which council can drag it out and confirm/deny whatever was on there. Otherwise yes, a vote should be made if it was not.
> 
> What? Why is a public vote being done on the game?

Probably to get the most reach, idk I aint council

**Comment by azzzertyy** [comment]
> > > The vote was on the server itself, not the discord. I remember voting on it myself (trust me bro), but nonetheless it /should/ be archived somewhere, in which council can drag it out and confirm/deny whatever was on there. Otherwise yes, a vote should be made if it was not.
> > 
> > 
> > What? Why is a public vote being done on the game?
> 
> Probably to get the most reach, idk I aint council

If anything I would argue that does the exact opposite of that. It limits the voting population to whoever is on the game currently rather than the wider playerbase. If this is true, council was being stupid in their decision making here. If so, I will still argue it should be a quickfire, but a council vote will be acceptable to, its just utterly insane that they would hold a vote on a small subset of the playerbase instead of the playerbase at large.

**Comment by adamsong** [comment]
The poll has not occurred, and even if it had occurred, it would not have included this, the strap was added by the dev team independent of the council votes surrounding batons

**Comment by azzzertyy** [comment]
> The poll has not occurred, and even if it had occurred, it would not have included this, the strap was added by the dev team independent of the council votes surrounding batons

so-im-right-logan-nelson

**Comment by Hislyna** [comment]
> > The poll has not occurred, and even if it had occurred, it would not have included this, the strap was added by the dev team independent of the council votes surrounding batons
> 
> so-im-right-logan-nelson

Yeah poll didn't occur when it shoud've, and instead dev team did their own shiz to balance batons sooooooooooooooooooooooooooo, fun!

File: code/__DEFINES/dcs/signals/signals_atom/signals_atom_movable.dm [hunk 1]
```diff
@@ -42,8 +42,6 @@
 #define COMSIG_MOVABLE_BUCKLE "buckle"
 ///from base of atom/movable/unbuckle_mob(): (mob, force)
 #define COMSIG_MOVABLE_UNBUCKLE "unbuckle"
-///from base of atom/movable/throw_at(): Triggers when throwing, before the item is dropped and before COMSIG_MOVABLE_PRE_THROW 
-#define COMSIG_MOVABLE_PRE_DROPTHROW "movable_pre_dropthrow"
```
Comment (tattax) [inline code/__DEFINES/dcs/signals/signals_atom/signals_atom_movable.dm]: why remove the signal?

Comment (azzzertyy) [reply code/__DEFINES/dcs/signals/signals_atom/signals_atom_movable.dm]: not needed i dont think
its not, not used by anything else

Comment (tattax) [reply code/__DEFINES/dcs/signals/signals_atom/signals_atom_movable.dm]: don't think it should be removed anyhow

Comment (azzzertyy) [reply code/__DEFINES/dcs/signals/signals_atom/signals_atom_movable.dm]: ill add it back

File: code/modules/mob/living/carbon/carbon.dm [hunk 2]
```diff
@@ -187,7 +187,6 @@
 
 	else if(!CHECK_BITFIELD(I.item_flags, ABSTRACT) && !HAS_TRAIT(I, TRAIT_NODROP))
 		thrown_thing = I
-		SEND_SIGNAL(thrown_thing, COMSIG_MOVABLE_PRE_DROPTHROW, src)
```
Comment (adamsong) [inline code/modules/mob/living/carbon/carbon.dm]: why remove the signal?

**Review by adamsong — CHANGES_REQUESTED** [review]

**Review by adamsong — APPROVED** [review]
Code does as described

Awaiting HD for merge

**Pull request merged**